src/utils/fileUpload.js

```diff
@@ -45,27 +45,19 @@ const findUniquePath = async function(client, userRoot, path) {
 	if (await client.exists(userRoot + path) === false) {
 		return path
 	}
+
 	// Get the file extension (if any)
 	const fileExtension = getFileExtension(path)
-	// If there's a file extention, remove it from the path string
-	if (fileExtension !== '') {
-		path = path.substring(0, path.length - fileExtension.length)
-	}
-	// Check if the path ends with ` (n)`
-	const suffix = path.match(/ \((\d+)\)$/) ? path.match(/ \((\d+)\)$/) : ''
```

Comment:
no longer checking for this?

So if I download `file (2).txt` and modify it and then upload it again, it turns into `file (2) (2).txt` instead of `file (3).txt`

Reply:
Yes, as I'm not sure this is the right way to do this.
- download + upload leads to creating a new file in the system, and we don't have mechanisms for checking, if `file (2).txt` in NC Files and `file (2).txt` from your PC is really the same but modified, or completely different.
- I can revert it, but this should be also considered when checking for duplicate names

Reply:
Well we surely don't want to have `file (2) (2).txt`

So yeah please revert for now

Reply:
Reverted for this PR